pkgs/servers/clickhouse/default.nix

```diff
@@ -36,6 +36,11 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     rm -rf $out/share/clickhouse-test
+
+    sed -i -e '/<log>\/var\/log\/clickhouse-server\/clickhouse-server\.log<\/log>/d' \
+      $out/etc/clickhouse-server/config.xml
```

Comment:
you can use `!` to delimite regex-parts to avoid escaping slashes in your paths.

Reply:
makes sense. I'm currently rebuilding this :)